src/System.IO/src/System/IO/Stream.cs

```diff
@@ -28,6 +28,36 @@ internal SemaphoreSlim EnsureAsyncActiveSemaphoreInitialized()
             // WaitHandle, we don't need to worry about Disposing it.
             return LazyInitializer.EnsureInitialized(ref _asyncActiveSemaphore, () => new SemaphoreSlim(1, 1));
         }
+        
+        // This method is used by CopyTo, CopyToAsync,
+        // and NullStream's override of the latter.
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
I'd really like to keep the code as similar as possible between the two repos; I realize they've already diverged, but I'd like to avoid further divergence as much as possible.